src/schema/resolvers/user-resolver.js

```diff
@@ -0,0 +1,13 @@
+import * as userController from '../../controllers/user-controller'
+
+export default {
+  Query: {
+    users: () => userController.findAllUsers(),
+    user: ({ id }) => userController.findUserById(id)
+  },
+
+  Mutation: {
+    createUser: (args) => userController.createUser(args),
+    deleteUser: (id) => userController.deleteUser(id)
+  }
+}
```

Comment:
:thinking: Did you test these resolvers? I don't think they'll work. Resolvers receive four parameters in this order:
1. `obj`: a parent object. For a Query or Mutation it's a global object defined when creating the Apollo Server instance.
1. `args`: the arguments passed to the Query or Mutation. This is where you receive the data you're using above.
1. `context`: a context object defined when creating the Apollo Server instance.
1. `info`: internal information about the received GraphQL query. Not useful for our purposes.

Also, you should be only mapping resolvers to functions fro, the controller here. The functions from the controller have to receive at least the first two or three parameters I described above:
```javascript
import * as userController from '../../controllers/user-controller'

export default {
  Query: {
    users: userController.findAllUsers,
    user: userController.findUserById
  },

  Mutation: {
    createUser: userController.createUser,
    deleteUser: userController.deleteUser
  }
}
```

In the controller:
```javascript
import models from '../models'

export const createUser = (_, { input }) => {
  return models.user.create(input)
}

export const deleteUser = (_, { id }) => {
  return models.user.destroy({
    where: { id }
  })
}

export const findAllUsers = () => {
  return models.user.findAll()
}

export const findUserById = (_, { id }) => {
  return models.user.findByPk(id)
}
```